circle/functions

```diff
@@ -128,6 +129,43 @@ install_google_cloud_sdk() {
   fi
 }
 
+install_azure_cli() {
+  if ! which az >/dev/null ; then
+    info "Installing Python3 and its dependencies..."
+    apk update
+    if ! which python >/dev/null; then
```

Comment:
It would be possible the machine already includes python but it is not python3, I would check python3 instead